pkg/strategy/rebalance/strategy.go

```diff
@@ -255,9 +256,9 @@ func (s *Strategy) generateOrder(ctx context.Context) (*types.SubmitOrder, error
 		}
 
 		if side == types.SideTypeBuy {
-			quantity = fixedpoint.Min(quantity, balances[s.QuoteCurrency].Available.Div(ticker.Sell))
+			quantity = fixedpoint.Min(quantity, balances[s.QuoteCurrency].Available.Div(ticker.Sell)).Round(market.VolumePrecision, fixedpoint.Down)
```

Comment:
I think you can use:

market.RoundDownQuantityByPrecision

and 

market.RoundUpQuantityByPrecision

Reply:
done